src/monet.d.ts

```diff
@@ -249,6 +249,7 @@ interface IEitherStatic extends IMonadStatic {
   unit: IRightStatic;
   of: IRightStatic;    // alias for unit
   pure: IRightStatic;  // alias for unit
+  fromTry<V>(fn: () => V) : Either<any,V>;
```

Comment:
it should return `Either<Error, V>` or give possibility to define possibble throwable, e.g.: `fromTry<V, E = ERROR>(…): Either<E, V>`